docs/memory.md

```diff
@@ -7,21 +7,28 @@ title: Using the Memory View
 
 ## What is it?
 
-Dynamically allocated Dart objects e.g., ```new ClassName()``` live in a portion of memory called the heap.
-
-DevTool’s Memory profile tab lets you peek at how an isolate is using memory at a given moment. This page also provides accumulator counts that you can use to study the rate of memory allocation. The accumulators are useful if you suspect that your application is leaking memory, or has other bugs relating to memory allocation.
-
-The memory view displays information about a Flutter application's memory usage as well the ability to detect memory leaks
-controlling the VMs allocation accumulator. Memory profiler consists of four parts, each increasing in granularity:
-- Memory usage overview chart
+Allocated Dart objects created using a class constructor see [Dart Using constructors](https://www.dartlang.org/guides/language/language-tour#using-constructors) e.g.,
+```dart
+new MyClass()
+```
+or
+```dart
+MyClass()
+```
+, live in a portion of memory called the heap.
+
+DevTools Memory tab lets you peek at how an isolate is using memory at a given moment. This pane, using Snapshot and Reset, can display accumulator counts.  The accumulators can be used to study the rate of memory allocations, if you suspect your application is leaking memory or has other bugs relating to memory allocation.
+
+Memory profiling consists of four parts, each increasing in granularity:
+- Memory overview chart
 - Event timeline
-- Classes allocated in the heap total instances, bytes allocated, accumulator of allocations since last reset
+- Classes allocated in the heap, total instances, bytes allocated, and an accumulator of allocations since the last reset
 - Instances of a class type
 
-**Use a profile build of your application to analyze performance.** Memory usages are not indicative of release
-performance unless your application is run in profile mode.
+**Use a profile build of your application to analyze performance.** Memory usage is not indicative of release
+performance unless your application is run in profile mode.  In general, memory usage is fairly accurate in relative terms in debug, release or profile mode.  However, the absolute memory used maybe higher in debug versus release builds. 
```

Comment:
Nit: add a sentence about why it is higher in debug builds. E.g. because work that is done ahead of time in a release build has to be done at runtime. (?)

Reply:
**Use a profile build of your application to analyze performance.** Memory usage is not indicative of release performance unless your application is run in profile mode. In general, memory usage is fairly accurate in relative terms between debug, release or profile mode. However, the absolute memory used maybe higher in debug versus release builds. In a release build work can be computed and optimized ahead of time.  However, in a debug build that same work may have to be computed at runtime.